IdentityCore/src/webview/embeddedWebview/challangeHandlers/ios/MSIDCertAuthHandler.m

```diff
@@ -92,7 +92,9 @@ + (BOOL)completeCertAuthChallenge:(NSURL *)endUrl
 
 + (void)resetHandler
 {
+#if TARGET_OS_IPHONE && !MSID_EXCLUDE_SYSTEMWV
```

Comment:
why we added this extra check for system webview here ?

Reply:
This is a suggested change from Teams based on testing with the system webview disabled. I can see that all other instances of `s_certAuthInProgress` being updated are wrapped in this macro